integrations/cassandra/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## 1.2.0 (TBD)
+### Added
+- Updated SDK to v3.
```

Comment:
that's not fully true. It's v3.0.1-beta

Reply:
Anyway I would mention `beta`.

Reply:
I'll set it to v2.0.0-beta, is it ok?

Reply:
Hm... Sorry, but now I'm not sure how it really should be. Maybe `beta` is also not a good idea, since the integration is already `GA`. But I don't know how we should mention that we use SDK version `beta`. 
/cc @areina